### PR TITLE
Display players choices

### PR DESCRIPTION
**Pull request by tjperr**

### CODE REVIEW

File: src/server/player.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export class Player {
     this.game = null;
     this.name = '';
     this.errorMessage = null;
+    this.score = 0;
```
Comment (weizhongtan) [inline src/server/player.js:12]: Think this should be stored against the game

File: src/server/subRound.js [hunk 2]
```diff
@@ -24,17 +24,21 @@ export class SubRound {
   }
 
   getCaptions() {
-    return this.captions;
+    return this.captions.map((caption) => {
+      return {
+        playerName: caption.player.name,
```
Comment (weizhongtan) [inline src/server/subRound.js:29]: We should use the `getName` method instead of getting name directly

Comment (tjperr) [reply src/server/subRound.js:31]: done

File: src/server/player.js (hunk 1 above)
Comment (tjperr) [reply src/server/player.js:12]: removing this for now as I'll do the scoring in another PR. But think that makes sense

**Comment by weizhongtan** [comment]
Just need some conflicts resolving

**Pull request merged**